lib/solana_rpc_ruby/methods_wrapper.rb

```diff
@@ -0,0 +1,109 @@
+require 'json'
+require 'pry'
+require_relative 'request_body'
+
+module SolanaRpcRuby
+  class MethodsWrapper
+    include RequestBody
+
+    attr_accessor :api_client
+
+    attr_accessor :cluster
```

Comment:
all attr_accessors can be in one line

Reply:
I am going to add some descriptions later, so I'll leave it as it is.